### PR TITLE
NavBar Updates

### PR DESCRIPTION
**Pull request by charlie-tf**
- update padding
- update background and scrollbar thumb styles
- add transparent background for empty states:
  - species, seedbanks, nurseries, planting-sites

color and padding around scrollbar:
image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -206,6 +219,22 @@ export default function App() {
     reloadSpecies();
   }, [reloadSpecies]);
 
+  const reloadTracking = useCallback(() => {
```
Comment (karthikbtf) [inline src/App.tsx:222]: are these from merging main into visual-updates?

**Review by karthikbtf — APPROVED** [review]
Thanks 🎉 finally!!! The navbar scrollbar squeeze has been a big pain.

File: src/App.tsx (hunk 1 above)
Comment (charlie-tf) [reply src/App.tsx:222]: No, I added those myself. I needed to add set the navbar background to transparent for the planting-sites page empty state, but if one is added I needed to trigger a refresh to set it to not be transparent (I'll need to do an update of the planting-sites list as currently it shows a background image, but shouldn't, when non-empty)

Comment (karthikbtf) [reply src/App.tsx:222]: Got it 👍

**Pull request merged**